packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -39,6 +39,9 @@ const Set<PointerDeviceKind> _kLongPressSelectionDevices = <PointerDeviceKind>{
   PointerDeviceKind.invertedStylus,
 };
 
+
+const double _kSelectableComparingThreshold = 5.0;
```

Comment:
This would likely break if there is more padding in widget span?

Reply:
Also this should probably call vertical threshold?

Reply:
renamed to _kSelectableVerticalComparingThreshold